chaos/experiments/leader-election.sh

```diff
@@ -132,7 +132,7 @@ check_leader() {
     # Check the response for specific strings to determine the node status
     if [[ "$response" == *"append_transaction_executions called on leader node"* ]]; then
         return 0 # Success exit code for leader
-    elif [[ "$response" == *"APPEND_SUCCESS"* ]]; then
```

Comment:
I think this should remain OR use a specific message, we actually could change this function to probe several orther behaviors

Reply:
I will have to improve these checks when I work on protobuf definitions soon. I will leave it to the next PR